### DISCUSSION TITLE
Service Error 412: SMTP: Greeting never received

### DISCUSSION THREAD

**Discussion by benepe** [emailjs-com/help#59] [Q&A] [answered]
Above mentioned service error is returned sporadically (s. exemplary event log below), sometimes twice a day, then no errors for a couple of weeks.
I have not been able to find a pattern.
Is this an issue on the SMTP server side?
Should I just try to send the mail again?
Thank you!

![image](https://github.com/emailjs-com/help/assets/2700379/33b92b1a-da4a-4715-9adb-5bae252dfb42)

**Comment by xr0master** [comment, marked as answer]
This means that our service was able to connect to your SMTP server, but did not receive a greeting from it (the initial command that the SMTP server is ready to work).

Most often, yes, this means that the SMTP server is not so stable or is not always available.
You may try to resend this email.
